[gamemodes]/[race]/race/race_errorcodes.lua

```
@@ -0,0 +1,30 @@
+errorCode = {}
+
+local function loadAdditionalErrorCodes()
+	-- Load error codes from votemanager resource if it's running
+	local votemanagerResource = getResourceFromName("votemanager")
+	if (votemanagerResource) then
+		if (getResourceState(votemanagerResource) == "running") then
+			local votemanagerErrorCodes = exports.votemanager:getErrorCodes()
+			if (votemanagerErrorCodes) and (type(votemanagerErrorCodes) == "table") then
+				for key, code in pairs(votemanagerErrorCodes) do
+					-- If the error code is not defined here already, then set it
+					if (errorCode[key] == nil) then
```

I think that having the ability for race to override keys *might* actually cause confusion in the future. I think instead of ignoring already defined codes, we should error out: `assert(errorCode[key] == nil)`

What do you think?

---

Sounds fair enough, I've changed this now.